anesthetic/plot.py

```diff
@@ -251,6 +252,26 @@ def fastkde_plot_1d(ax, data, *args, **kwargs):
         lower/upper prior bound
         optional, default None
 
+    fill: Bool
+        fills between iso probability confidence regions (68%, 95%) if True.
+        Default False.
+
+    levels: list
+        amount of mass within each iso-probability lines.
```

Comment:
Grammar is a bit off here -- perhaps drop 'each'.

The full stops are also quite inconsistent, although I think this is endemic to a lot of the code, but it's nice for additions to be consistent (perhaps raise an issue for this, which could form a general 'docstring improvement' suggestion place).